% Generated by roxygen2: do not edit by hand
% Please edit documentation in R/functions.R
\name{.stop}
\alias{.stop}
\title{Ponownie rzuca przekazanym wyjatkiem zachowujac jego pierwotne wywolanie}
\usage{
.stop(e)
}
\arguments{
\item{e}{wyjatek przechwycony funkcja tryCatch()}
}
\value{
void
}
\description{
_
}
\details{
_
}
